shared/actions/__tests__/provision.test.js

```diff
@@ -89,7 +89,7 @@ describe('provisioningManagerProvisioning', () => {
       expect(response.result).not.toHaveBeenCalled()
       expect(response.error).toHaveBeenCalledWith({
         code: RPCTypes.constantsStatusCode.scinputcanceled,
-        desc: Constants.cancelDesc,
+        desc: 'Input canceled',
```

Comment:
Can we keep this as a constant?

Reply:
i think its ok for this test to know about the value. i don't want to export the individual constants out

Reply:
Oh I misread this as the main action file. Sounds good